packages/clerk-js/src/ui/primitives/Td.tsx

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
 
-import { PrimitiveProps } from '../styledSystem';
+import { createVariants, PrimitiveProps, StyleVariants } from '../styledSystem';
 import { Box, BoxProps } from './Box';
 
-export type TdProps = PrimitiveProps<'td'> & Omit<BoxProps, 'as'>;
+const { applyVariants, filterProps } = createVariants(theme => ({
+  base: {
+    fontSize: theme.fontSizes.$xs,
+    fontWeight: theme.fontWeights.$normal,
+    color: theme.colors.$blackAlpha700,
```

Comment:
This should be `color: theme.colors.$colorText` in order to respect the color from `appearance.variables`. The figma effect can be achieved with `opacity`.

(same applies to all other places)

Reply:
✔️